src/components/clusters/Clusters/Clusters.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback, useMemo } from 'react';
+import { Route, Routes, useLocation } from 'react-router-dom';
+
+import { Tab, Tabs, TabTitleText } from '@patternfly/react-core';
+
+import { Navigate, useNavigate } from '~/common/routing';
+import { AppPage } from '~/components/App/AppPage';
+
+import ClusterList from '../ClusterListMultiRegion';
+import ClusterTransferList from '../ClusterTransfer/ClusterTransferList';
+
+import { ClustersPageHeader } from './ClustersPageHeader';
+
+export const Clusters = () => {
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  const activeTabKey = useMemo(() => {
+    const path = location.pathname;
+
+    if (path.endsWith('/list') || path.endsWith('/list/')) return 'list';
+    if (path.endsWith('/requests') || path.endsWith('/requests/')) return 'requests';
+
+    return 'list';
+  }, [location.pathname]);
+
+  const handleTabSelect = useCallback(
+    (_event: React.MouseEvent<HTMLElement>, tabKey: string | number) => {
+      const targetPath = tabKey === 'requests' ? '/clusters/requests' : '/clusters/list';
```

Comment:
The paths for the subroutes are hardcoded multiple times in this file. It's better to define them once in an object and reuse the same values everywhere, something like:

```
const CLUSTERS_ROUTES = {
  LIST: '/clusters/list',
  REQUESTS: '/clusters/requests',
};
```

Reply:
I like it, I will make the change.